.babelrc

```diff
@@ -1,4 +1,5 @@
 {
+  "plugins": ["transform-es2015-modules-umd"],
```

Comment:
no, that's the wrong place. See https://github.com/babel/babel-preset-env#modules